doc/api/next_api_changes/2018-06-27-AL.rst

```diff
@@ -1,6 +1,28 @@
 Changes to backend loading
 ``````````````````````````
 
+It is now possible to set ``rcParams["backend"]`` to a *list* of candidate
+backends.
+
+If `.pyplot` has already been imported, Matplotlib will try to load each
```

Comment:
By "`.pyplot` has already been imported" you mean `matplotlib.use()` hasn't been called?  Does pyplot fundamentally have anything to do with this, other than its where 99% of us call `matplotlib.use()` implicitly?